homeassistant/components/apcupsd/strings.json

```diff
@@ -0,0 +1,20 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
+      "no_status": "No status is reported from [%key:common::config_flow::data::host%]"
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
+    },
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "port": "[%key:common::config_flow::data::port%]"
+        },
+        "description": "Enter the [%key:common::config_flow::data::host%] and [%key:common::config_flow::data::port%] on which the apcupsd NIS is being served."
```

Comment:
We don't mix in references inside sentences as that may not work well for all languages.